src/main/java/org/havenapp/main/ui/CameraViewHolder.java

```diff
@@ -280,43 +276,25 @@ public void updateCamera ()
             mDecodeWorkQueue);
 
     // A queue of Runnables
-    private final BlockingQueue<Runnable> mEncodeVideoWorkQueue = new LinkedBlockingQueue<Runnable>();
+    private final BlockingQueue<Runnable> mEncodeWorkQueue = new LinkedBlockingQueue<Runnable>();
```

Comment:
explanation: like mEncodeVideoThreadpool, this is renamed to not mention video, since video encoding is done inside CameraView

Reply:
Variable name makes it harder to read the code and know what this is encoding, video/audio/image